Countdown/Countdown.js

```diff
@@ -17,6 +17,7 @@ if (widgetInputRAW !== null) {
   }
   dateForCountdown = widgetInput[0].trim()
   icon = widgetInput[1] || '⏳';
+  text = widgetInput[3]; // new
```

Comment:
Please make this "optional" to avoid breaking changes: Use "Countdown" as fallback value in case there is no text defined